src/main/java/com/sumindaa/demo/employeeservice/controller/EmployeeController.java

```diff
@@ -0,0 +1,13 @@
+package com.sumindaa.demo.employeeservice.controller;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+class EmployeeController {
```

Comment:
as mentioned in #2 we should have REST endpoints for all CRUD operations
CRUD, Create, Read, Update, Delete

Reply:
Done. However still I didn't do the service method and will do soon